src/main/java/sh/damon/stackmob/mixin/LivingEntityMixin.java

```diff
@@ -26,16 +26,15 @@ public void onDeath(DamageSource source, CallbackInfo info) {
         sm.entityManager.unregisterStackedEntity(stackEntity);
 
         int size = stackEntity.getSize();
-        if (size == 1) return;
+        if (size <= 1) return;
```

Comment:
in welke case kan dit lager zijn dan 1? of is het al 0 op moment dat de laatste sterft

Reply:
I've only had this occur in cases where the mod might fuck up and spawn an entity of size 0, this prevents a stack entity that can be killed 4 billion times before it dies (underflow check just in case).